### ISSUE TITLE
Make aliases for quickbuy

### ISSUE THREAD

**Issue by falkecarlsen** [f-klubben/stregsystemet#260]
```
As a fember

I would like to be able to say `username classic` to purchase a Tuborg Classic

so that I don't have to remember the product id 14.
```

This is solved in stregsystem-cli by way of first checking if product-input is in shorthand, otherwise try matching for product id: https://github.com/f-klubben/stregsystemet-cli/blob/master/main.py#L25-L53

I suspect this would require a larger rewrite of the very old parser and a new handwritten EBNF for the new language. I don't think this feature should be implemented if it diminishes our error pointer and EBNF printing on input error, because it's a cool aesthetic to have.

**Comment by jones-g** [comment]
We previously discussed decoupling purchase identifier from database table ID which should fix this issue. Our suggestion was to create a SKU that references a product. Purchases should then be "username SKU". This would also allow to have multiple SKUs for the same product - and not be limited by a number if need be. That would support "famle" members like me to still purchase a 14 or 32 and newer fembers could purchase "classis" or "coffee". That might require less of a refactor.

When we migrated the stregsystem to github I actually created a issue that has this kind of idea here https://github.com/f-klubben/stregsystemet/issues/35
